utils/generate-docs-examples/README.md

```diff
@@ -0,0 +1,14 @@
+This directory contains a script that generates the Python documentation examples in `docs/examples`.
+
+To use this script you need a recent version of Node.js (18+). First install the dependencies:
```

Comment:
The request converter supports the three current LTS releases, 18, 20 and 22. I think it is best to let the user choose one instead of making a choice with an .nvmrc file.